torch/csrc/autograd/python_variable.cpp

```diff
@@ -630,7 +630,7 @@ static PyObject* THPVariable_make_wrapper_subclass(
       "int64_t? storage_offset=None, MemoryFormat? memory_format=None, ScalarType dtype=None, "
       "Layout layout=torch.strided, Device device=None, bool pin_memory=False, bool requires_grad=False, "
       "c10::string_view? dispatch_sizes_strides_policy=None, bool dispatch_device=False, bool dispatch_layout=False)",
-      "_make_wrapper_subclass(PyObject* cls, SymIntArrayRef size, SymIntArrayRef strides, "
+      "_make_wrapper_subclass(PyObject* cls, SymIntArrayRef size, SymIntArrayRef? strides=None, "
```

Comment:
This change looks like it could have been done on its own, with a test on its own.

Reply:
Yep this could have been its own PR, sorry. In fact, I'm going to remove this from the PR and not try to change this schema.

One annoying thing I ran into while working on this PR was that `make_wrapper_subclass` technically has two overloads, but the first will happily specialize on any symints, while the second won't. The second schema is a bit more annoying to use (strides are required, no kwargs). It seems like eventually, we probably want to either consolidate the two overloads, or just have the first properly support dynamic shapes - what do you think? I didn't want to handle that though so far in my PR(s), so I'm just working around it.